code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -15,9 +16,21 @@
 	var/heating_power=40000
 	var/list/autolink = null
 
+	var/datum/wires/transmitter/wires = null
+
 	var/const/RADS_PER_TICK=150
 	var/const/MAX_TEMP=70 // Celsius
-	machine_flags = MULTITOOL_MENU
+	machine_flags = MULTITOOL_MENU | SCREWTOGGLE
+
+/obj/machinery/media/transmitter/broadcast/New()
+	..()
+	wires = new(src)
+
+/obj/machinery/media/transmitter/broadcast/Destroy()
+	if(wires)
+		wires.Destroy()
```

Comment:
You didn't add a destroy proc for the wire datum, add one to null the juke reference